### DISCUSSION TITLE
Welcome to ChatWizard Discussions!

### DISCUSSION THREAD

**Discussion by lisiur** [lisiur/ChatWizard#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TheEmperix** [comment]
Hello, I can't understand, is the app paid or not? 
And what is the price in the bottom left corner? 
Thank you for the answer.

**Reply by lisiur** [reply]
Hi! This application is free; however, it requires an OpenAI key which need be paid for.

[OpenAI keys](https://platform.openai.com/account/api-keys)
